Update vipps-login-migrate-api-1.0-to-2.0.md (#46)
vippsas/vipps-login-api: vipps-login-migrate-api-1.0-to-2.0.md

Vipps Login will be the first service to integrate with this new endpoint.

## API changes

> User information will no longer be available as part of the 'id_token'. The 'id_token' will return the ID for the user ('sub'), this needs to be used to get the additional user information from the '/userinfo' endpoint. See here for an example of the [userinfo response](https://vippsas.github.io/vipps-login-api/#/public/userinfo).

Reviewer: ```suggestion
User information will no longer be available as part of the 'id_token'. The 'id_token' will return the ID for the user ('sub'). To get the additional user information a request has to be sent to the '/userinfo' endpoint. See here for an example of the [userinfo response](https://vippsas.github.io/vipps-login-api/#/public/userinfo).
```
